src/main/java/user/createCorporatePage/PageGenerateInteractor.java

```diff
@@ -0,0 +1,80 @@
+package user.createCorporatePage;
+import entity.CorporatePage;
+import entity.Page;
+import service.IPageGenerateDataAccess;
+import service.PageGenerateDataAccess;
+import service.ServerStatus;
+import user.createCorporatePage.exceptions.PageCreationFailed;
+
+public class PageGenerateInteractor implements IPageGenerateInput {
+
+    private final IPageGenerateDataAccess dataAccess;
+//    private final PageGenerateOutputBoundary pageGenerateOutputBoundary;
+    private final PageFactory pageFactory;
+
+    public PageGenerateInteractor(IPageGenerateDataAccess dataAccess, PageFactory pageFactory){
+        this.dataAccess = dataAccess;
+//        this.pageGenerateOutputBoundary = pageGenerateOutputBoundary;
+        this.pageFactory = pageFactory;
+    }
+
+    public PageGenerateResponseModel create(PageGenerateRequestModel requestModel){
+
+//        if (pageGenerateDsGateway.existsByPageLabel(requestModel.getInputLabel())){
```

Comment:
Same as above, just explain why this is commented out.

Reply:
comment deleted as of now